packages/fxa-settings/src/assets/images/apple_app_store_button/da.svg

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I know I said this on Slack already but yeah be sure to send all these suckers through [svgo](https://github.com/svg/svgo) 😎